index.js

```diff
@@ -15,17 +15,10 @@ function submitTodo() {
 };
 
 function toggleTodo() { 
-  
-  this.parentElement.style.textDecoration = this.checked ? "line-through" : "";
-
-  if (this.parentElement.style.textDecoration == "line-through" && this.parentElement.childNodes[2] == undefined) {
 
-    const button = document.createElement("button");
-    button.innerHTML = "remove";
-    this.parentElement.appendChild(button);
+  todos[this.parentElement.id].done = this.checked ? true : false;
```

Comment:
You can do this simpler like this, `todos[this.parentElement.id].done = this.checked`